web/src/components/Plugin/locales/en-US.ts

```diff
@@ -120,6 +123,8 @@ export default {
   'component.pluginForm.limit-count.count.tooltip': 'The specified number of requests threshold.',
   'component.pluginForm.limit-count.time_window.tooltip':
     'The time window in seconds before the request count is reset.',
+  'component.pluginForm.limit-count.key_type.tooltip':
+    'The key type, support: "var" (single var) and "var_combination" (combine var)',
   'component.pluginForm.limit-count.key.tooltip':
     'The user specified key to limit the count, now accept those as key: "remote_addr"(client\'s IP), "server_addr"(server\'s IP), "X-Forwarded-For/X-Real-IP" in request header, "consumer_name"(consumer\'s username) and "service_id".',
```

Comment:
Now the key is `Input` not `Select`, so we can just keep `The user specified key to limit the count.` for `component.pluginForm.limit-count.key.tooltip`

Reply:
removed

Reply:
thanks~